src/index.html

```diff
@@ -1,19 +1,384 @@
 <!doctype html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1.0"
-    />
-    <title>Dia</title>
-    <link
-      rel="stylesheet"
-      href="./styles/main.scss"
-    />
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Air</title>
+    <link rel="icon" type="image/x-iconn"
+    href="images/logo/Air.png">
+    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Poppins:wght@600&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Dia</h1>
-    <script src="scripts/main.js"></script>
+
+    <header class="header">
+      <div class="header__icons">
+        <a href="#">
+          <img src="images/logo/Air.png" alt="logo">
+        </a>
+        <a href="#menu" class="icon icon--menu"></a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#about-us" class="nav__link">About us</a></li>
+          <li class="nav__item"><a href="#services" class="nav__link">Services</a></li>
+          <li class="nav__item"><a href="#testimonials" class="nav__link">Testimonials</a></li>
+          <li class="nav__item"><a href="#hire-us" class="nav__link">Contact us</a></li>
+        </ul>
+      </nav>
+      <a href="#hire-us" class="nav__hire-us">Hire us</a>
+    </header>
+
+    <aside class="menu" id="menu">
+      <div class="menu__icons">
+        <a href="#">
+          <img src="images/logo/Air.png" alt="logo">
+        </a>
+        <a href="#" class="icon icon--close"></a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#about-us" class="nav__link">About us</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">home</a></li>
+          <li class="nav__item"><a href="#who-we-are" class="nav__link">Work</a></li>
+          <li class="nav__item"><a href="#services" class="nav__link">Services</a></li>
+          <li class="nav__item"><a href="#hire-us" class="nav__link">Contact us</a></li>
+        </ul>
+      </nav>
+    </aside>
+
+    <section class="strategic-agency" id="about-us">
+      <div class="strategic-agency--top">
+        <h1 class="strategic-agency--top-title">Strategic Agency</h1>
+        <span class="strategic-agency--top-content">We believe in the power of bold ideas that can solve business challenges.</span>
+        <button class="btn" href="#">Learn more</button>
+      </div>
+
+      <div class="strategic-agency--bottom">
+        <div class="carousel">
```

Comment:
This carousel does not seem to be working properly on 1440p screens [Reference](https://github.com/mate-academy/layout_dia#:~:text=Large%20screens%202560px)

![image](https://github.com/mate-academy/layout_dia/assets/52002856/04195a86-0e43-47c7-97a0-47ca1b860255)

Reply:
I'm aware of that, you just have to reload the page and it should work fine.. I used JS on this project.